bin/benchmark.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+java -cp "$(cd `dirname $0`/..; pwd)"'/tools/benchmark-cli/build/libs/benchmark-cli.jar:*' \
```

Comment:
@jakelandis let's add an issue for doing this for the ingest-converter + this tool? I agree with you 100% here though, but I think it's worth documenting this move properly (just so we know what happened when we get some manifest error in a release late at night or something :D).